entries/h/hpe.com.json

```
@@ -0,0 +1,20 @@
+{
+  "HPE": {
+    "domain": "hpe.com",
+    "tfa": [
+      "totp",
+      "custom-hardware",
+      "custom-software"
+    ],
+    "custom-hardware": [
+      "Biometrics"
+    ],
+    "custom-software": [
+      "Okta Verify"
+    ],
+    "documentation": "https://support.hpe.com/hpesc/public/docDisplay?docId=sd00002926en_us&page=GUID-78722BEA-6517-4C6E-96F3-92E9D81903E2.html",
+    "categories": [
+      "other"
```

```suggestion
      "cloud",
      "hosting"
```
I feel like these are the most appropriate categories.

---

Thank you very much! 👍